AirCasting/SearchAndFollow/AirBeamMeasurements/AirBeamStreamPrefix.swift

```diff
@@ -13,5 +13,12 @@ enum AirBeamStreamPrefix: CaseIterable {
         case .airBeam3: return "airbeam3"
         }
     }
+    
+    var capitalizedName: String {
```

Comment:
I wouldn't call it `capitalizedName`, rather something in the realm of `userFacingName`